similar/main/bm.cpp

```diff
@@ -640,12 +646,33 @@ static grs_bitmap *bm_load_extra_objbitmap(const char *name)
 	}
 }
 
+static void bm_unload_last_objbitmaps(int count)
```

Comment:
Generally, counters should be `unsigned`, because it does not make sense to free a negative quantity of something.